### PR TITLE
fix(curriculum): intro for basic HTML review page

### LINKED ISSUE #56866: [ Full Stack Cert ] - update description for basic HTML review page
Let’s update the intro text to the following:

```md
        "intro": ["Before you are quizzed on the HTML knowledge you have gained so far, you first need to review the concepts. Open up this review page to see which concepts you need to review."]

```

Here is the file you need to edit

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/i18n/locales/english/intro.json#L1739-L1742

### PR DESCRIPTION
**Pull request by AdityaInnovates**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56866

### CODE REVIEW

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Comment by AdityaInnovates** [comment]
Could you please add the "hacktoberfest-accepted" label to this PR?

**Review by gikf — COMMENTED** [review]
Hey @AdityaInnovates,

There's no need for additional label in the PR. Whole repository is labelled as opting-in for Hacktoberfest.

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -1738,7 +1738,7 @@
       },
       "review-basic-html": {
         "title": "Basic HTML Review",
-        "intro": ["Review the basic HTML topics."]
+        "intro": ["Before you are quizzed on the HTML knowledge you have gained so far, you first need to review the concepts. Open up this review page to see which concepts you need to review."]
```
Comment (gikf) [inline client/i18n/locales/english/intro.json:1741]:
```suggestion
        "intro": [
          "Before you are quizzed on the HTML knowledge you have gained so far, you first need to review the concepts. Open up this review page to see which concepts you need to review."
        ]
```
This should make prettier happy.

**Review by gikf — APPROVED** [review]

**Comment by gikf** [comment]
Hi @AdityaInnovates. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**